src/ec/packages/ec-component-site-header/site-header.test.js

```diff
@@ -0,0 +1,59 @@
+import path from 'path';
+import { renderTwigFile } from '@ecl-twig/test-utils';
+
+describe('EC - Site Header', () => {
+  const template = path.resolve(__dirname, './site-header.html.twig');
+  const defaultIconPath = 'static/icons.svg';
+
+  describe('Site Header default', () => {
+    test(`- renders correctly`, done => {
+      expect.assertions(1);
+
+      const data = {
+        icon_file_path: defaultIconPath,
+        header_link: {
```

Comment:
we should have values for the english version too (currently test snaps are almost empty)

Reply:
Fixed in https://github.com/ec-europa/ecl-twig/pull/33/commits/db86360dccd7fd6ec6e71d44a13560fda4930945